cmd/release-tool/changelog.go

```diff
@@ -135,6 +135,20 @@ func getChangelog(gqlClient *github.GQLClient, repo string, branch string, tag s
 	return changeloggenerator.New(config.repo, commitInfos)
 }
 
+func getCommitPR(commit *github.GQLCommit) *github.GQLPRNode {
+	if len(commit.AssociatedPullRequests.Nodes) == 0 {
+		return nil
+	}
+
+	for _, prNode := range commit.AssociatedPullRequests.Nodes {
+		if prNode.Merged && strings.Contains(commit.Message, fmt.Sprintf("(#%d)", prNode.Number)) {
```

Comment:
```suggestion
		if prNode.Merged {
```

While it's very likely I'm not sure relying on commit message is the right way to go.

PRs have the data on where they landed as a commit sha right? Why don't we use this?

Reply:
I can try to use it. It does have that property `mergeCommit`, however I didn't test how it behaves when PRs are merged using different ways (squash/merge/rebase).